presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -212,6 +212,9 @@
 
     //TODO: Prestissimo related session properties that are temporarily put here. They will be relocated in the future
     public static final String PRESTISSIMO_SIMPLIFIED_EXPRESSION_EVALUATION_ENABLED = "simplified_expression_evaluation_enabled";
+    public static final String SMART_SAMPLING_ENABLED = "smart_sampling_enabled";
```

Comment:
Generally speaking, I prefer a more descriptive session variable name. Maybe something like "partition_based_sampling_enabled".

Reply:
Yeah but it's so.. smart! lol will fix it.

Reply:
I will call it key_based_sampling, partition is too loaded and unclear